docs/internal/input-validations-framework-readme.md

```diff
@@ -0,0 +1,56 @@
+# Description
+
+This is an internal readme to explain how to use the validation constraints framework to detect
+errors at build time rather than run time.
+
+## What can you use with the framework
+
+You will find in the constraints-utils.ts file the generic calls for the following classes if you properly import them from the constraints-utils.ts file.
+
+## generic class of validations constraints-utils.ts
```

Comment:
Which class are you referring to here? It is file. You can refer to it as "constraints framework" or "constraints module".

Reply:
I meant like the following things as in the rest of the read me, I guess that was not very clear. Would listing it out be okay like StringConstarint, UrlStringConstraint, NumberConstraint, ArrayConstraint, and the validateConstraints function

Reply:
you can call it something like "supported contraints":

Reply:
Oops just changed what was below that, changed it to `Validations file constraints-utils.ts`